packages/next-codemod/bin/upgrade.ts

```diff
@@ -410,19 +410,37 @@ function isUsingAppDir(projectPath: string): boolean {
  * 3. Otherwise, we ask the user to manually add `--turbopack` to their dev command,
  *    showing the current dev command as the initial value.
  */
-async function suggestTurbopack(packageJson: any): Promise<void> {
+async function suggestTurbopack(
+  packageJson: any,
+  targetNextVersion: string
+): Promise<void> {
   const devScript: string = packageJson.scripts['dev']
+  // Turbopack flag was changed from `--turbo` to `--turbopack` in v15.0.1-canary.3
+  // PR: https://github.com/vercel/next.js/pull/71657
+  // Release: https://github.com/vercel/next.js/releases/tag/v15.0.1-canary.3
+  const isAfterTurbopackFlagChange =
+    compareVersions(targetNextVersion, '15.0.1-canary.3') >= 0
+  const turboPackFlag = isAfterTurbopackFlagChange ? '--turbopack' : '--turbo'
 
   if (!devScript) {
     console.log(
-      `${pc.red('⨯')} Could not find a "dev" script in your package.json.`
+      `${pc.yellow('⚠')} No "dev" script found in your package.json. Skipping Turbopack suggestion.`
```

Comment:
this feels more like output I'd expect to see with a `--debug` flag rather being in the common case. Not sure if others agree but I'd probably just remove it all together.

Reply:
IMHO, it's rare for an existing app to be missing `scripts.dev`, and shouldn't we not silence it when it isn't applicable? I agree that it's a bit verbose, but it would be better to inform users correctly that something wasn't as expected!

Reply:
Maybe it's just the language. I'm viewing it as someone who doesn't know what Turbopack is. The warning would make me think "what is a Turbopack suggestion? Why should I care?"

Maybe the language should be more about suggesting the user add a dev script to help with local development?

Reply:
Agreed on omitting `skipping turbopack suggestion` as it's not super relevant in the context

Reply:
The context should include Turbopack somehow because without it we don't even need to inform that "dev" script is missing.

Reply:
I can already see the feedback: "Why didn't it ask me to switch to turbopack?".

How about 

```suggestion
      `No "dev" script found in your package.json. If you're not already using Turbopack, consider enabling it by using `next dev --turbopack`: https://nextjs.org/docs/architecture/turbopack.`
```